.config/.lintstagedrc.js

```diff
@@ -9,6 +9,8 @@ export default {
 	// And elsewhere we don't, compare to https://github.com/stylelint/stylelint/pull/8009
 	'*.{css,scss}': 'stylelint --fix --allow-empty-input --no-validate',
 	'*.{js,ts,tsx,jsx,mjs,cjs}': 'xo --fix',
-	// ensure that security vulnerabilities are fixed before committing
-	'package-lock.json': 'npm audit fix --omit=dev'
+	// ensure that security vulnerabilities are fixed before committing - we need to skip `dev` for the moment as there are some unsolveable conflicts
+	'package-lock.json': 'npm audit fix --omit=dev',
+	// ensure that lock file is up to date
+	'package.json': 'npm install --package-lock-only --ignore-scripts'
```

Comment:
```suggestion	
	'**/package.json': () => 'npm install --package-lock-only --ignore-scripts',
```
lint-staged passes matched files as args. Therefore, we should use a function so all package.json files (args) are handled.